### PR TITLE
data: usmangt

### PR DESCRIPTION
**Pull request by usmangt**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

An excellent Open Source app I need to start for my personal work instead using Linktree. I really appreciate your hard work on this. Great job!! 

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6837"></a>

### CODE REVIEW

**Review by HassanTanveer — DISMISSED** [review]

**Review by harshsingh32 — COMMENTED** [review]
looks good

File: data/usmangt.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+		"Cloud (SaaS)",
+		"Database",
+		"Technical Support"
+	],
+
+
+	"links": [{
+			"name": "Follow me on LinkedIn",
+			"url": "https://www.linkedin.com/in/syed-usman-ahmad-b1415515/",
+			"icon": "FaLinkedIn"
```
Comment (kumarsonsoff3) [inline data/usmangt.json:23]: You probably be using an invalid icon here. It should be `FaLinkedin`

```suggestion
			"icon": "FaLinkedin"
```

File: data/usmangt.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+		},
+		{
+			"name": "Follow me on GitHub",
+			"url": "https://github.com/usmangt",
+			"icon": "FaGitHub"
```
Comment (kumarsonsoff3) [inline data/usmangt.json:28]: Likewise here `FaGithub`.

```suggestion
			"icon": "FaGithub"
```

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Review by kumarsonsoff3 — APPROVED** [review]

**Pull request merged**